### ISSUE TITLE
Plugin causes app icon issues. It overrides the app icon we define in our application.

### ISSUE THREAD

**Issue by karan-rawal** [hathibelagal-dev/launcher-assist-for-flutter#2]

**Comment by shinewanna** [comment]
Yes, author or someone should fix, it's useless coz of this overriding app icon.

**Comment by shinewanna** [comment]
ok,i have been work around with the author code.
The author builded fixed icon in his plugin, the name is ic_launcher.png(the default icon name).
You can solved with changing the icon name in your manifest and mipmap of your project.
